cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5023,6 +5023,14 @@
     </description>
   </property>
 
+  <property>
+    <name>support.bundle.temp.dir</name>
+    <value>${local.data.dir}/tmp</value>
```

Comment:
nit: Since this is specific to support bundle, should we call the directory `supportBundleData` instead of `tmp`?